examples/bri-3/components/identity/bpiSubjectAccount/bpiSubjectAccount.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { IBpiSubject } from "../bpiSubject/bpiSubject.interface";
+
+export interface IBpiSubjectAccount {
+    id: string; // TODO: Add uuid after #491
+    creatorBpiSubject: IBpiSubject;
+    ownerBpiSubject: IBpiSubject;
+    bpiAccounts: string[]; // TODO: Add as part of #489
```

Comment:
should that be not `bpiAccounts: bpiAccount[];`?

Reply:
We do not have BpiAccounts at the moment -they will be implemented as part of #489